katdal/chunkstore_s3.py

```diff
@@ -699,7 +702,8 @@ def _read_xml(response):
 
     def mark_complete(self, array_name):
         """See the docstring of :meth:`ChunkStore.mark_complete`."""
-        self.create_array(array_name)
+        bucket = self.split(array_name, 1)[0]
```

Comment:
Thanks! You could have just used this line in `create_array`, but I like the explicit `_create_bucket`.